app/scripts/views/complete_reset_password.js

```diff
@@ -74,7 +74,7 @@ define(function (require, exports, module) {
         isLinkDamaged: ! doesLinkValidate,
         isLinkExpired: isLinkExpired,
         isLinkValid: doesLinkValidate && ! isLinkExpired,
-        isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled()
+      //  isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled()
```

Comment:
This line should be complete removed from the source code. Remove the comma on the previous line as well.